### ISSUE TITLE
Neo4j instance

### ISSUE THREAD

**Issue by mihaic** [tudelft-atlarge/graphalytics-platforms-neo4j#1]
Make clear in the documentation the fact that we use a Neo4j instance downloaded by Maven which we distribute in our JAR.

Document a way to use an alternative instance of Neo4j.

**Comment by thegeman** [comment]
As far as I am aware, it is not possible to use the Neo4j Java API with a non-local instance (i.e. any Neo4j instance running in a different JVM), so adding support for those would require new implementations of the algorithms in e.g. Cypher. According to Neo4j's devs, the Java API is the most performant, so for fair comparison to other platforms we need to keep supporting our current implementations.

In short: documentation will be added, I will also describe how a different Neo4j version could be compiled against and packaged, but support for external Neo4j instances is a won't fix unless contributed by others.

**Comment by szarnyasg** [comment]
See also PR #6 - using the graph algorithm library will allow users to run algorithms remotely and efficiently at the same time.
